CoreGraphFramework/src/au/gov/asd/tac/constellation/graph/locking/ValueStore.java

```diff
@@ -67,41 +67,41 @@ public void setObject(final Object value) {
     }
 
     public final void save() {
-        converter.save();
+        converter.save(access);
     }
 
-    private static interface Saver extends Serializable {
-
-        public void save();
+    private interface Saver extends Serializable {
+        
+        public void save(ParameterWriteAccess PWA);
```

Comment:
Can these be a `final` parameter?

Reply:
Yeah, sorry I keep forgetting to do that

Reply:
Also an all uppercase variable name like that is not good Java style.